### ISSUE TITLE
Same hostnames but different system names appears only once on Ansible groups 

### ISSUE THREAD

**Issue by lucecca** [cobbler/ansible#2]
I'm facing a problem (but not sure if what I'm facing is the designed behaviour) integrating Cobbler (Cobbler 3.3.2) and Ansible (2.9.27) while accessing the Cobbler dynamic inventory. I've configured in cobbler 2 systems with different names but same hostname T (`cobbler system add --name wp05-15 --profile prof1; cobbler system add --name wp05-20 --profile prof1; cobbler system edit --name wp05-15 -hostname wp05;  cobbler system edit --name wp05-20 -hostname wp05;` ). Whenever I access the dynamic inventory via `ansible-inventory --list`, ansible displays only 1 host named wp05 in every automagically generated group. 
I've even tried enabling the option `allow_duplicate_hostnames` on `/etc/cobbler/settings.yaml` but no joy ...

**Comment by SchoolGuy** [comment]
Due to https://github.com/cobbler/ansible/issues/1#issuecomment-1004505673 I would kindly ask you to open the issue [here](https://github.com/ansible-collections/community.general/issues). You may ping me directly. I will on the weekend then take the time as promised and take a look. I will also archive this repository on the weekend when doing so. :)

**Comment by SchoolGuy** [comment]
I will close this issue in this repository. I will open a new issue on behalf of you in the proper one.
